sencha-workspace/packages/slate-cbl/src/view/student/DashboardController.js

```diff
@@ -0,0 +1,137 @@
+/*jslint browser: true, undef: true *//*global Ext,Slate*/
+Ext.define('Slate.cbl.view.student.DashboardController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.slate-cbl-student-dashboard',
+    requires: [
+        'Slate.cbl.model.Student',
+        'Slate.cbl.model.Competency',
+        'Slate.cbl.view.student.skill.OverviewWindow'
+    ],
+
+    config: {
+        id: 'slate-cbl-teacher-dashboard', // workaround for http://www.sencha.com/forum/showthread.php?290043-5.0.1-destroying-a-view-with-ViewController-attached-disables-listen-..-handlers
+        control: {
+            '#': {
+//                contentareachange: 'refresh',
```

Comment:
The contentareachange event is commented out because I got lazy in crunch time and opted for reloading the whole page when switching between content areas instead of supporting dynamic transitions

The progressrowclick event below is commented out because the "Recent Progress" table didn't make the cut for the first release